common/src/test/templates/io.netty.util.collection/KObjectHashMapTest.template

```diff
@@ -69,69 +69,72 @@ public boolean equals(Object obj) {
         }
     }
 
-    private IntObjectHashMap<Value> map;
+    private @K@ObjectHashMap<Value> map;
 
     @Before
     public void setup() {
-        map = new IntObjectHashMap<Value>();
+        map = new @K@ObjectHashMap<Value>();
     }
 
     @Test
     public void putNewMappingShouldSucceed() {
         Value v = new Value("v");
-        assertNull(map.put(1, v));
+        @k@ key = 1;
+        assertNull(map.put(key, v));
         assertEquals(1, map.size());
-        assertTrue(map.containsKey(1));
+        assertTrue(map.containsKey(key));
         assertTrue(map.containsValue(v));
-        assertEquals(v, map.get(1));
+        assertEquals(v, map.get(key));
     }
 
     @Test
     public void putShouldReplaceValue() {
         Value v1 = new Value("v1");
-        assertNull(map.put(1, v1));
+        @k@ key = 1;
+        assertNull(map.put(key, v1));
 
         // Replace the value.
         Value v2 = new Value("v2");
-        assertSame(v1, map.put(1, v2));
+        assertSame(v1, map.put(key, v2));
 
         assertEquals(1, map.size());
-        assertTrue(map.containsKey(1));
+        assertTrue(map.containsKey(key));
         assertTrue(map.containsValue(v2));
-        assertEquals(v2, map.get(1));
+        assertEquals(v2, map.get(key));
     }
 
     @Test
     public void putShouldGrowMap() {
-        for (int i = 0; i < 10000; ++i) {
-            Value v = new Value(Integer.toString(i));
-            assertNull(map.put(i, v));
-            assertEquals(i + 1, map.size());
-            assertTrue(map.containsKey(i));
+        for (@k@ key = 0; key < 255; ++key) {
```

Comment:
Do we support `byte` as `@k@`?  If so I think we have to cast `255` with `(@k@) 255`.